### ISSUE TITLE
Terminal navigation mode broken

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#107574]
Reported by @saschacowley

"I got the update to 1.49.2 today, and it seems that it has broken terminal navigation. When I use Ctrl + Up/Down, I am only able to read to the top of the visible console output."

I tried this on my macOS and also does not work for me. So it seems like a regression.
